tests/core/test_snapshot.py

```diff
@@ -399,7 +399,7 @@ def test_fingerprint_seed_model():
 
     expected_fingerprint = SnapshotFingerprint(
         data_hash="3834815287",
-        metadata_hash="1120323454",
+        metadata_hash="1624292983",
```

Comment:
This is due to correctly including the batch_size now in the metadata hash for seeds.